package.json

```diff
@@ -1,15 +1,25 @@
 {
   "name": "zigbee-adapter",
-  "version": "0.3.0-pre2",
+  "version": "0.3.0",
   "description": "Zigbee adapter plugin for Mozilla IoT Gateway",
   "main": "index.js",
+  "scripts": {
+    "lint": "eslint ."
+  },
   "keywords": [
     "mozilla",
     "iot",
     "adapter",
     "zigbee"
   ],
   "homepage": "https://github.com/mozilla-iot/zigbee-adapter",
+  "dependencies": {
+    "color": "^3.0.0"
+  },
+  "devDependencies": {
+    "eslint": "^4.7.0",
+    "eslint-plugin-html": "^4.0.0"
+  },
   "license": "MPL-2.0",
   "repository": {
     "type": "git",
```

Comment:
You should probably add `node_modules` to the `files` array.

Reply:
Won't that just check that the node_modules directory exists?
If we were going to go all in, it probably makes sense to run sha256sum on each file in the node_modules directory.

Reply:
And fix package.sh to remove node_modules and run yarn install --production so that we only include what's needed for the adapter (i.e. remove the devDependencies)

Reply:
The simple `node_modules` check is at least a safeguard that the add-on was packaged properly, but yeah, you could go all-in as well.